package.json

```diff
@@ -47,6 +47,7 @@
     "@fullhuman/postcss-purgecss": "1.3.0",
     "@mdx-js/loader": "0.18.0",
     "@svgr/webpack": "5.5.0",
+    "@swc/cli": "0.1.49",
```

Comment:
where is swc/cli been used?

Reply:
It's useful for local testing of spack during benchmarking 👍